README.md

```diff
@@ -62,6 +62,21 @@ $ station metrics --follow
 ...
 ```
 
+### `$ station activity`
+
+Get Station activity logs.
```

Comment:
In #47, I am adding the following text to each command that's reading state files:

```md
_This command reads information from the state files created by the main Station
process._
```

Let's do the same here, too, please.

Reply:
Will adjust once https://github.com/filecoin-station/core/pull/47#discussion_r1135351332 is resolved

Reply:
Looks like no change is necesasry